applications/GeoMechanicsApplication/tests/test_settlement_workflow/python_workflow/Kratos_stages.py

```diff
@@ -5,17 +5,11 @@
 
 if __name__ == "__main__":
 
-    currentWorking = os.getcwd()
-
     # construct parameterfile names of stages to run
-    project_path = r"C:\Users\graaf\OneDrive - Stichting Deltares\Documents\settlement_app\scenario_1\input_files"
     n_stages = 4
-    parameter_file_names = [os.path.join(project_path, 'ProjectParameters_stage' + str(i + 1) + '.json') for i in
+    parameter_file_names = ['ProjectParameters_stage' + str(i + 1) + '.json' for i in
                             range(n_stages)]
```

Comment:
We can simplify this even further:
```suggestion
    parameter_file_names = [f'ProjectParameters_stage{i+1}.json' for i in range(n_stages)]
```